src/server.ts

```diff
@@ -35,31 +43,40 @@ const listenMessage = `Listening on port: ${config.APP_PORT}. ENV is ${config.EN
  * 5. Get movie by ID
  * */
 app.post("/movies", async (request: Request, response: Response) => {
+  const client = await MongoClient.connect(DB_URL);
+  const db = client.db("moviesDB");
+  const collection = db.collection("movies");
+
   const { name, comment, personalScore } = request.body;
   const data = await OmdbService.getMovie(name);
 
   // movie not found
   if (!data["imdbID"]) {
     // movie not found in IMDB - let's add it to local DB
-    const movie = { imdbID: movieStore.nextId(), Title: name, comment, personalScore };
-    movieStore.add(movie);
+    const movie = {
+      imdbID: movieStore.nextId(),
+      Title: name,
+      comment,
+      personalScore,
+    };
+    collection.insert(movie);
     return response.status(200).json({
       message: `New movie added: ${name}`,
       data: movie,
     });
   }
 
   // movie already exists in DB
-  if (movieStore.exist(data["imdbID"])) {
+  if (collection.findOne({ imdbID: data["imdbID"] })) {
     return response.status(200).json({
       message: `Movie already exist in DB: ${name}`,
     });
   }
 
   // movie is found in IMDB but not in local DB - so let's add it
-  if (!movieStore.exist(data["imdbID"])) {
+  if (!collection.findOne({ imdbID: data["imdbID"] })) {
     const movie = { ...data, comment, personalScore };
-    movieStore.add(movie);
+    collection.insert(movie);
```

Comment:
This one need `await` operator. And you can see warning in your logs:
[MONGODB DRIVER] Warning: collection.insert is deprecated. Use insertOne, insertMany or bulkWrite instead.
(Use `node --trace-warnings ...` to show where the warning was created)